Watchman.DomainModel/Responses/Commands/Handlers/RemoveResponseCommandHandler.cs

```diff
@@ -18,11 +18,9 @@ public async Task HandleAsync(RemoveResponseCommand command)
             using var session = _sessionFactory.Create();
             var onEvent = session.Get<Response>()
                 .FirstOrDefault(x => x.ServerId == command.ServerId && x.OnEvent == command.OnEvent);
-
+            
             if (onEvent == null)
-            {
                 return;
-            }
```

Comment:
Klamry zostaw, nawet jak jedna linijka, to przyjęliśmy w projekcie że zawsze robimy